packages/safe-apps-sdk/src/types/sdk.ts

```diff
@@ -79,3 +83,7 @@ export type AddressBookItem = {
   chainId: string;
   name: string;
 };
+
+export const isObjectEIP712TypedData = (obj?: unknown): obj is EIP712TypedData => {
```

Comment:
Is it possible to narrow the type of `obj` down as its only being used for `string | EIP712TypedData`?

Reply:
Not sure if I understand this correctly. Currently, the type guard can work with a variable of any type, but if we narrow it down, we will limit its useability. I don't see why it is better. Let me know if I miss something

Reply:
Thats a good point thanks! With `unknown` the guard can be used for more cases so no need to be as strict here